lib/mysqlDaoQueryHelper.js

```diff
@@ -25,6 +25,8 @@ class MysqlDaoQueryHelper {
             lte: '<=',
             eq: '=',
             ne: '!=',
+            in: 'IN',
+            nin: 'NOT IN',
```

Comment:
FYI, it's going to be not_in instead of notIn.  Cleaning code ends up snake casing the operator and I figured it'd be easier not to deal with updating that.